regress/regress1/CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+
+marabou_add_coav_test(1 "reluBenchmark1.30941200256s_UNSAT.nnet" UNSAT) 
+marabou_add_coav_test(1 "reluBenchmark0.94518494606s_SAT.nnet" SAT) 
```

Comment:
How did we pick these specific ones? Because they terminate quickly?

Reply:
I remember answering this but don't find the answer now...
Yes I am not familiar with the different networks, so I choose everything by the termination time.
In regress0, the fastest example I found per case (SAT and UNSAT for every network type)
In regess1, Some acasxu networks, most of coav (since they are pretty fast) 
and In regress2 some more of acasxu networks.

Can you think on something clever to do?